packages/components/src/components/brandshape/brandshape.stories.ts

```diff
@@ -79,13 +79,21 @@ export const Variants = {
       axis: {
         y: {
           type: 'attribute',
-          name: 'variant'
+          name: 'variant',
```

Comment:
the image variant seems to have the image not centered. In this story, the placeholder image icon is slightly visible. Can we center the image in the background? 

Not sure about if it should be centered by default but at least in this story to make it look less "broken".

Reply:
![CleanShot 2024-07-08 at 09 52 53](https://github.com/solid-design-system/solid/assets/26542182/70f115c0-a5db-46ce-be11-b7bdf3137369)

Just tested it and looks not correct? We definitely need a story which actually shows an image.

<img
  class="fit-picture"
  src="https://interactive-examples.mdn.mozilla.net/media/cc0-images/grapefruit-slice-332-332.jpg"
  alt="Grapefruit slice atop a pile of other slices" />

Reply:
At least we should document, how this should be implemented.

Reply:
@mariohamann @karlbaumhauer I opted to use custom transform styles to adjust images. This was necessary as different images have different focus points. Please see "Sample: Positioning Image Variant" as that is where I described the general idea + gave an example. Then we can resolve this thread if this solution looks good to you :)

Reply:
fine for me.